components/application-authenticators/org.wso2.carbon.identity.application.authenticator.basicauth/src/main/java/org/wso2/carbon/identity/application/authenticator/basicauth/BasicAuthenticator.java

```diff
@@ -189,18 +192,24 @@ protected void processAuthenticationResponse(HttpServletRequest request,
             if (userRealm != null) {
                 userStoreManager = (UserStoreManager) userRealm.getUserStoreManager();
                 isAuthenticated = userStoreManager.authenticate(MultitenantUtils.getTenantAwareUsername(username), password);
+                if(isAuthenticated){
+                    audit.info(String.format(AUDIT_MESSAGE, username, "SUCCESS"));
+                }
             } else {
+                audit.info(String.format(AUDIT_MESSAGE, username, "FAILURE"));
                 throw new AuthenticationFailedException("Cannot find the user realm for the given tenant: " + tenantId);
             }
         } catch (IdentityRuntimeException e) {
             if(log.isDebugEnabled()){
                 log.debug("BasicAuthentication failed while trying to get the tenant ID of the user " + username, e);
             }
+            audit.info(String.format(AUDIT_MESSAGE, username, "FAILURE"));
```

Comment:
Let's remove the audit logs from BasicAuthenticator because log in the framework, so it will get logged twice. But then in framework we don't log for error cases. We have to think about it further. Logging in each authenticator may not be scalable. So for now let's remove it.